website/server/extractDocs.js

```diff
@@ -54,18 +54,48 @@ function getPlatformFromPath(filepath) {
   return CROSS_SUFFIX;
 }
 
-function getExample(componentName, componentPlatform) {
-  var path = '../Examples/UIExplorer/' + componentName + 'Example.js';
-  if (!fs.existsSync(path)) {
-    path = '../Examples/UIExplorer/' + componentName + 'Example.'+ componentPlatform +'.js';
-    if (!fs.existsSync(path)) {
-      return;
+function getExamplePaths(componentName, componentPlatform) {
+  var componentExample = '../Examples/UIExplorer/' + componentName + 'Example.';
+  var pathsToCheck = [
+    componentExample + 'js',
+    componentExample + componentPlatform + '.js',
+  ];
+  if (componentPlatform === CROSS_SUFFIX) {
+    pathsToCheck.push(
+      componentExample + IOS_SUFFIX + '.js',
+      componentExample + ANDROID_SUFFIX + '.js'
+    );
+  }
+  var paths = [];
+  pathsToCheck.map((p) => {
+    if (fs.existsSync(p)) {
+      paths.push(p);
     }
+  });
+  return paths;
+}
+
+function getExamples(componentName, componentPlatform) {
+  var paths = getExamplePaths(componentName, componentPlatform);
+  if (paths) {
+    var examples = [];
+    paths.map((p) => {
+      var platform = p.match(/Example\.(.*)\.js$/);
+      var title =
+        (componentPlatform === CROSS_SUFFIX) &&
```

Comment:
This form of `string = boolean && boolean && string` is a bit hard for my brain :)
I would make it an explicit assignment and not abuse coercion.